app/assets/javascripts/Components/Result/url_viewer.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import "../../../stylesheets/common/_url_viewer.scss";
+
+export class URLViewer extends React.Component {
+  state = {
+    url: "https://www.youtube.com/embed/dQw4w9WgXcQ", // Sample Youtube embed link
+    //url: "https://drive.google.com/file/d/1yQh8bzo4IzmtSt-qZ0Vb-BzzEXsp45v_/preview" // sample google drive embed link
+    //url: "https://play.library.utoronto.ca/watch/4493e554354a7e51bdf90f3197185b5f" // sample mymedia link
+  };
+
+  componentDidMount() {
```

Comment:
You can just use the prop as the default value for state if you're setting state in the constructor:

```js
  constructor(props) {
    super(props);
    this.state = {
      url: this.props.url,
    };
  }
```

Reply:
However, I don't think you even need to set the state at all here. Just use the prop directly:

```js
        <iframe className="url-display" src={this.props.url} allowFullScreen />
```

You don't need to update this value inside this component.